static/js/components/ThumbnailList.jsx

```diff
@@ -111,6 +111,12 @@ const Thumbnail = ({ ra, dec, name, url, size, grayscale }) => {
             className={imgClasses}
             title={alt}
             loading="lazy"
+            onError={(e) => {
+              if (url !== "#") {
+                e.target.onerror = null;
```

Comment:
```suggestion
```
Unless there's a reason that's needed?

Reply:
Well, I guess it's fine, if there's ever an error rendering the fallback image, this might prevent an infinite loop